{{ cookiecutter.repo_name }}/setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
```

Comment:
Basically from reading new PEP's it's better to use and store the meta data in a config file and run it through a setup.py. I believe it's because you want to a clean setup.py file as this is what is being run, therefore making any possible security issues more obvious. https://ianhopkinson.org.uk/2022/02/understanding-setup-py-setup-cfg-and-pyproject-toml-in-python/

Also, having only a setup.py file means the setup.py file is mainly just listing configurable options, which is not the best code quality. However, by adding a setup.cfg file, the setup.py file becomes a much cleaner and more inline with our best practice. https://towardsdatascience.com/setuptools-python-571e7d5500f2#:~:text=cfg%20file%20may%20be%20more,file%20in%20the%20new%20setup.

As for documentation, I'll have to amend any refences to setup/file structure but the install process is the same and has been added.